apps/livre/Livre.cpp

```diff
@@ -69,13 +69,21 @@ int main( const int argc, char** argv )
     }
 
     // 2. initialization of local client node
-    livre::ClientPtr client = new livre::Client( );
+    livre::ClientPtr client = new livre::Client();
+    if( !client->parseArguments( argc, argv ))
+    {
+        eq::exit();
+        // cppcheck-suppress unreachableCode
+        livre::exitErrors();
+        return EXIT_SUCCESS;
```

Comment:
Yes and no. If help or version, then (obviously) not, otherwise maybe. What now?